lib/cocoapods/generator/copy_resources_script.rb

```diff
@@ -100,6 +100,7 @@ def script
 set -e
 
 mkdir -p "${CONFIGURATION_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}"
+mkdir -p "${PODS_ROOT}/resources-checksums"
```

Comment:
Yep you are right its more "derived datas"

Like said md5 hash is just to check that the source file changed and need be re-compiled. I think the build file presence have to be checked at all.